gallery/tutorials/pipeline_demo.py

```diff
@@ -67,19 +68,13 @@ def download(url, save_path, chunk_size=1024 * 1024):
 # %%
 # Noise and CTF Filters
 # ^^^^^^^^^^^^^^^^^^^^^
-# Let's start by creating noise and CTF filters. The ``operators`` package contains a collection
-# of filter classes that can be supplied to a ``Simulation``. We use ``ScalarFilter`` to create
-# Gaussian white noise and ``RadialCTFFilter`` to generate a set of CTF filters with various defocus values.
+# Let's start by creating CTF filters. The ``operators`` package contains a collection
```

Comment:
Maybe drop "Noise and" from the title of this section.

Reply:
yep